sys/net/gnrc/pktdump/gnrc_pktdump.c

```diff
@@ -91,7 +91,7 @@ static void _dump_snip(gnrc_pktsnip_t *pkt)
 #ifdef MODULE_CCN_LITE_UTILS
         case GNRC_NETTYPE_CCN_CHUNK:
             printf("GNRC_NETTYPE_CCN_CHUNK (%i)\n", pkt->type);
-            printf("Content is: %.*s\n", pkt->size, (char*)pkt->data);
+            printf("Content is: %.*s\n", (int)pkt->size, (char*)pkt->data);
```

Comment:
Just dumping data as a string seems like a bad idea, but at least it is bound :-). Anyway, not related to this PR...